### PR TITLE
feat(types): add Typescript types

### PR DESCRIPTION
**Pull request by iGoodie**
Amazing wrapper! Saves me tons of time, thanks for taking your time and creating it.

I needed to create the Types for my project. I thought why not share it, so everybody else can also use it with Typescript :)

### CODE REVIEW

**Comment by futurify-ydang** [comment]
Will the library's owner merge this PR? I also want to use TypeScript with this package.

**Comment by alexvilchis** [comment]
I also want typings!

File: types.d.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+declare module "react-barcode" {
+  declare namespace ReactBarcode {
```
Comment (orhanna14) [inline types.d.ts:2]: remove `declare` below module, 
TS error: A 'declare' modifier cannot be used in an already ambient context.

Comment (iGoodie) [reply types.d.ts:2]: That's more elegant. I'll change namespace declaration to a module one!
I'm still so new to Typescript. Thanks for letting me know!

**Pull request merged**